Mathlib/RingTheory/Coprime/Lemmas.lean

```diff
@@ -151,8 +151,9 @@ theorem exists_sum_eq_one_iff_pairwise_coprime [DecidableEq I] (h : t.Nonempty)
       use fun i ↦ if i = a then u else v * μ i
       have hμ' : (∑ i ∈ t, v * ((μ i * ∏ j ∈ t \ {i}, s j) * s a)) = v * s a := by
         rw [← mul_sum, ← sum_mul, hμ, one_mul]
-      rw [sum_cons, cons_eq_insert, sdiff_singleton_eq_erase, erase_insert hat, if_pos rfl,
-        ← huv, ← hμ', sum_congr rfl]
+      rw [sum_cons, cons_eq_insert, sdiff_singleton_eq_erase, erase_insert hat]
+      simp only [↓reduceIte, ite_mul]
+      rw [← huv, ← hμ', sum_congr rfl]
```

Comment:
This fails on `nightly-testing`, I haven't actually diagnosed why.

Reply:
Happy to take this diff out (or for anyone else to do so).

Reply:
Whichever you think is most convenient...